src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -99,6 +99,9 @@ protected class OrderLocations : Comparer<LSP.Location>
 
         protected virtual TestComposition Composition => s_composition;
 
+        private protected virtual TestAnalyzerReferenceByLanguage TestAnalyzerReferences
```

Comment:
moved diagnostics initialization into the base so that individual tests don't have to do something special to get diagnostics working.